packages/patternfly-4/src/content/get-started/about-pf.md

```diff
@@ -2,43 +2,40 @@
 path: "/get-started/about"
 ---
 # About PatternFly 4
-PatternFly is an open source design system that was created to enable consistency and usability across a wide range of applications and use cases. PatternFly provides clear standards, guidance, and tools to help designers and developers work together more efficiently to build better products and user experiences.
+PatternFly is an open source design system that was created to enable consistency and usability across a wide range of applications and use cases. PatternFly provides clear standards, guidance, and tools to help designers and developers work together more efficiently to build better user experiences.
 
 ## Basic structure
-PatterFly 4 is separated into isolated and modular structures that fall into two categories: layouts and components.
+
+### Components
+Components, like buttons and alerts, can be assembled together to build applications.
 
 ### Layouts
 Layouts are generic tools that allow you to structure and organize the content on your pages.
-[View layouts](URL)
 
-### Components
-Components, like buttons and alerts, can be assembled together to build applications.
-[View Core components](/documentation/core/) | [View React components](/documentation/react/)
+### Demos
+Demos use components and layouts in combination to show you how to build more complex structures and application views.
+
+[View components, layouts, and demos in HTML/CSS](/documentation/core) | [View components, layouts, and demos in React](/documentation/react)
 
 ## Style and design
```

Comment:
Should we align this with the way that the website is now organized? So we have Design Guidelines as a top level category and under that Styles, Usage and Behavior, and Content.